packages/components/src/color-picker/test/index.js

```diff
@@ -119,7 +119,7 @@ describe( 'ColorPicker', () => {
 			h: 125,
 			s: 0.2,
 			l: 0.5,
-			// add alpha to prove it's ignored
+			// add alpha to prove it's ignored.
```

Comment:
Sometimes seems that did not got the uppercase rule 🤔

Reply:
Ahhh this is because `a` is an identifier on the next line, and the rule will not enforce it if the comment starts with any identifiers on the next or previous lines. I'll make it check for an identifier and a space, that will solve this.

Reply:
Fixed in 0b0145f28a545c65c28c9a74bde5155de8c8090a